civo/kubernetes/resource_kubernetes_cluster.go

```diff
@@ -505,6 +515,13 @@ func resourceKubernetesClusterDelete(ctx context.Context, d *schema.ResourceData
 
 func customizeDiffKubernetesCluster(ctx context.Context, d *schema.ResourceDiff, meta interface{}) error {
 	if d.Id() != "" {
```

Comment:
Not related to this change, but the check is getting rather lengthy and it is making harder to review. We should invert this check and early return instead.

```go
	if d.Id() == "" {
		return errors.New("some error") //
	}
	
	if d.HasCHange("write_kubeconfig") {
		...
```